### PR TITLE
RestAPI destination

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: restapi/restapi.go [hunk 1]
```diff
@@ -292,3 +339,111 @@ func (r *RestAPI) getSalesforceOauthToken() (string, error) {
 
 	return authResponse.AccessToken, nil
 }
+
+func (r *RestAPI) setDefaults() {
+	if r.ListMethod == "" {
+		r.ListMethod = r.Method
+	}
+	if r.ListMethod == "" {
```
Comment (Baggerone) [inline restapi/restapi.go:347]: This is the same `if` statement as the one above.

File: restapi/restapi.go [hunk 2]
```diff
@@ -292,3 +339,111 @@ func (r *RestAPI) getSalesforceOauthToken() (string, error) {
...
+func (r *RestAPI) updateContact(p psync.Person, n *uint64, wg *sync.WaitGroup, eventLog chan<- psync.EventLogItem) {
+	wg.Done()
+}
+
+func (r *RestAPI) deleteContact(p psync.Person, n *uint64, wg *sync.WaitGroup, eventLog chan<- psync.EventLogItem) {
+	wg.Done()
+}
+
+func (r *RestAPI) httpRequest(verb, url, body string, headers map[string]string) (string,
+	error) {
```
Comment (Baggerone) [inline restapi/restapi.go:406]: Does this need to be split out to a new line like this?

File: restapi/restapi.go (hunk 1 above)
Comment (Schparky) [reply restapi/restapi.go:347]: Yes, it looks strange, so I added some comments.

File: restapi/restapi.go (hunk 2 above)
Comment (Schparky) [reply restapi/restapi.go:406]: Nope, thanks.

File: restapi/restapi.go (hunk 1 above)
Comment (Baggerone) [reply restapi/restapi.go:349]: Sorry. I hadn't looked inside the blocks carefully.

Comment (Schparky) [reply restapi/restapi.go:349]: No problem!

**Pull request merged**